docs/source/contribute/dev-assisted-package-creation.rst

```diff
@@ -0,0 +1,113 @@
+Developer Assisted Package Creation
+===================================
+
+If you are a user, developer, or researcher that has software related to geospatial data
+processing and are unable to integrate that functionality into GeoIPS, you've come to
+the right place. Some of our developers for GeoIPS (specifically those that are CIRA-based), might be
+able to help you integrate your functionality into GeoIPS. Please continue reading if
+you think this may be of use!
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as standalone packages that can interact with
+GeoIPS. This can be as simple as adding your algorithm or reader to the main GeoIPS
+code, or it can be more complex in creating a completely separate GeoIPS plugin-package
+specific for your use case.
+
+Please feel free to view any of our
+`public repositories <https://github.com/orgs/NRLMMD-GEOIPS/repositories>`_ for a
+general idea of what we can help you with. For any standalone package that we'll be
+developing for users who have requested assistance, we'll largely be following the
+structure of the `template_basic_plugin package <https://github.com/NRLMMD-GEOIPS/template_basic_plugin>`_.
+The aforementioned package is a template for creating separate plugin packages that can
+interact with GeoIPS seamlessly.
+
+The purpose of integrating software like this is to elevate GeoIPS as a package that can
+process nearly any type of satellite, in-situ, geo-located -derived data, and output it
+in a variety of formats. For each standalone package that we create, we only elevate the
+capabilities of GeoIPS as a whole. Eventually, we'd like to see GeoIPS be one, if not
+the package you need to handle your geospatial processing routines.
+
+Funding Requirements for Complex Transitions
+--------------------------------------------
+
+While we welcome all conversations about integrating external packages into GeoIPS, please note that if transitioning
+your package requires substantial development time and resources, we may require funding to support that effort. The
+complexity of the integration, the state of your existing codebase, and the extent of modifications needed will all
+factor into this assessment. We encourage you to reach out to discuss your specific needs, and we can provide guidance
+on potential resource requirements and funding options.
+
+Use Cases That We'll Support
+----------------------------
+
+If the data routines you've created don't match any of the repositories linked above,
+then we've likely found a use case worth creating a separate GeoIPS package for.
```

Comment:
What does it mean for data routines to not match the linked repositories?

Should we add information about what kinds of plugins we'd be interested to actually incorporate into core GeoIPS?

I'm wondering whether we should stipulate that anything that we help with without funding must be released open source.

Reply:
More explicit would be "don't match any of the _functionality implemented in_ any of the repos linked above". Changing